### PR TITLE
To do interactive list

### PR DESCRIPTION
**Pull request by ntzwilly**
# To Do list: interactive list
## Project requirements

- Add 2 new JavaScript files and import them as modules:
    - one will contain methods related to drag/drop and sorting functionality.
    - the other will contain methods related to the status updates (completed: true/false).
- Add event listeners to list items (dragstart, dragover, drop) and to checkbox (change).
- Update items object's values for index and completed keys upon user actions.
- Store the updated array of items in local storage, so the user gets the correct list values after the page reloads.

### CODE REVIEW

File: src/interactive.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  const input = elementGenerator('input', 'label', null, null);
+  input.value = elem.description;
+  const image = elementGenerator('img', 'more', null, null);
+  image.src = moreIcon;
+
+  if (elem.checked) {
+    input.classList.add('line-through');
+  } else {
+    input.classList.remove('line-through');
+  }
```
Comment (mricanho) [inline src/interactive.js]: - [x] Take out this status check on another file, the new file should contain methods related to the status updates (completed: true / false)

File: src/index.js [hunk 2]
```diff
@@ -1,9 +1,10 @@
+/* eslint-disable import/no-cycle */
 import './style.css';
-import moreIcon from './more.svg';
 import Icon from './enter.svg';
 import recycle from './recycle.svg';
+import { dragAndDrop } from './interactive.js';
```
Comment (mricanho) [inline src/index.js:6]: - [x] You need to import here the new file (Status check)

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi @ntzwilly, 

Good job so far! Your app it's already working, your JS code it's readable and clean, just a minor detail with modularity, you are gonna need that for the next milestone🔥🔥

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

yougotit

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/interactive.js (hunk 1 above)
Comment (ntzwilly) [reply src/interactive.js]: Thank you, I have now made the required changes

File: src/index.js (hunk 2 above)
Comment (ntzwilly) [reply src/index.js:6]: It is done

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @ntzwilly. 

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**